src/electron/index.ts

```diff
@@ -35,6 +34,17 @@ import {ShadowsocksLibevBadvpnTunnel} from './sslibev_badvpn_tunnel';
 import {TunnelStore, SerializableTunnel} from './tunnel_store';
 import {VpnTunnel} from './vpn_tunnel';
 
+// Build-time macros injected by webpack's DefinePlugin:
+//   - NETWORK_STACK is either 'go' or 'libevbadvpn' by default
+//   - SENTRY_DSN is either undefined or a url string
+//   - APP_VERSION should always be a string
+declare const NETWORK_STACK: string;
```

Comment:
i wonder if these might make more sense in a declaration file, not unlike `preload.d.ts`

Reply:
Sounds good, but maybe not in `preload.d.ts`, I would rather call it `webpack-electron-globals.d.ts`.

Reply:
I like the idea of having them defined somewhere in the actual code, with default values. Then they get overridden according to the build mode. But perhaps they belong in libraries, instead of index.ts. I'd punt on that change for now though. Limit the changes.

Reply:
Worth putting a `TODO` comment, at least